content/en/blog/2021-03-19-a-parrot-trainer-eats-crow.md

```diff
@@ -0,0 +1,100 @@
+---
+title: "A Parrot Trainer Eats Crow"
+slug: "a-parrot-trainer-eats-crow"
+date: "2021-03-19T08:42:21-04:00"
+draft: false
+image_webp: images/blog/parrots.webp
+image: images/blog/parrots.jpg
+author: Rebecca Bilbro
+description : "Models trained on massive datasets using millions of parameters can both have low bias and also be very biased - what can we in the ML community do about this?"
+---
+
+In this post, we'll consider how it is that models trained on massive datasets using millions of parameters can be both "low bias" and also *very biased*, and begin to think through what we in the ML community might be able to do about it.
+
+## Birds on a Wire
+
+In the machine learning community, we are trained to think of size as inversely proportional to bias. We associate small datasets with the problem of underfit, which is to say high bias. We learn that in the face of unfamiliar data, underfit models make poor assumptions that lead to inaccuracies. Likewise, we call models with smaller sets of hyperparameters "weak learners" because their limited complexity limits our ability to reduce bias even as our dataset size grows.
```

Comment:
It might be worth it to link the text "size as inversely proportional to bias" to Scott Fortman Roe's Bias vs Variance post -- to help our google foo with even more links to web content.

Reply:
great idea!